source/etc/machine.d/mast.l.cfg

```diff
@@ -12,6 +12,8 @@ export UDA_PORT2=${UDA_PORT}
 
 DATA_ARCHIVE=/net/raidsrvr/data
 
+export UDA_BYTES_PLUGIN_ALLOWED_PATHS="/net/raidsrvr/data,/common/uda-scratch,/projects/physics/omfit"
```

Comment:
should this be ; separated list rather than , separated?

Reply:
either or, as long as its not a '/' we're good right. 
Will change to ';', just for you ;)